CSharp/Microsoft.Bot.Schema.Teams/Generated/MessageActionsPayloadBody.cs

```diff
@@ -57,5 +60,12 @@ public MessageActionsPayloadBody(string contentType = default(string), string co
         [JsonProperty(PropertyName = "content")]
         public string Content { get; set; }
 
+        /// <summary>
```

Comment:
Do not modify the data model. Text context is something you want only in SDK I am guessing, it should not be in the DTO.

Reply:
fixed